<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="group_a">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>360</width>
      <height>280</height>
     </rect>
    </property>
    <property name="title">
     <string>Camera A</string>
    </property>
    <widget class="QLabel" name="camera_a">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>360</width>
       <height>260</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="group_b">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>360</width>
      <height>280</height>
     </rect>
    </property>
    <property name="title">
     <string>Camera B</string>
    </property>
    <widget class="QLabel" name="camera_b">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>360</width>
       <height>260</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="button_stop_start">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>520</y>
      <width>361</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>Parar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_processed">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>90</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Processadas:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_tuned">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>120</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Viradas:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_discarded">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>150</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Descartadas:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_cork">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>60</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Rolhas</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_quality">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>210</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Qualidade</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_average">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>240</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Media:</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_processed">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>90</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_turned">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>120</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_discarded">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>150</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_minimum">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>270</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Minima:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_maximum">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>300</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Maxima</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="quality_average">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>240</y>
      <width>118</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="value">
     <number>75</number>
    </property>
   </widget>
   <widget class="QToolButton" name="button_settings">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>20</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="quality_minimum">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>270</y>
      <width>118</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="value">
     <number>50</number>
    </property>
   </widget>
   <widget class="QProgressBar" name="quality_maximum">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>300</y>
      <width>118</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="value">
     <number>90</number>
    </property>
   </widget>
   <widget class="Line" name="separator">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>180</y>
      <width>351</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_selected">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>460</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <underline>true</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string>Rolha A</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_defect">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>350</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Defeito</string>
    </property>
   </widget>
   <widget class="Line" name="separator_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>330</y>
      <width>351</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_defect_a">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>380</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Rolha A:</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_defect_a">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>380</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_defect_b">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>410</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_defect_b">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>410</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Rolha B:</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
